guacamole/pom.xml

```diff
@@ -493,6 +493,13 @@
             </exclusions>
 
         </dependency>
+        
+        <!-- JSTZ for TimeZone Detection -->
```

Comment:
This will need to be added to the list of bundled dependencies within `LICENSE`.